plugins/module_utils/openssh/certificate.py

```diff
@@ -326,8 +336,17 @@ def cert_type(self, cert_type):
     def signing_key_fingerprint(self):
         return fingerprint(self.signing_key)
 
-    @abc.abstractmethod
     def public_key_fingerprint(self):
+        if self.public_key_type_string is None:
+            return b''
```

Comment:
Returning `b''` instead of `None` is also an API breakage, I would add a docstring that specifies that `None` will change to `b''` in community.crypto 3.0.0.

Reply:
`b''` is actually consistent with the existing behavior.`self.public_key_type_string` is a new property added with this refactor which is why there is a `None` check now.